ThirdParty/Bolts/BoltsInClasspath/README.md

```diff
@@ -0,0 +1,81 @@
+﻿The sample uses openweathermap.org free subscription rest API to demonstrate the Bolts usage. Please register for free and use the token generated from https://home.openweathermap.org/users/sign_up and replace it with apiKey placeholder in the Constants.m file
+
+This project uses bolts as a dependency.When you first clone the repo if you have included ‘--recursive’ skip the next statement. Else
+	Navigate to samples root directory and run the ‘git submodule update --init --recursive’ git command.
+
+Now Make sure you are using the Bolts 1.8.4 version submodule which has hash
+	e64deecb2f0e10ac0dbb71f522c7a5b9cafb0b4d
+	(you can do that by navigating to the WinObjC-Samples/ThirdParty/Bolts/BoltsInClasspath directory in cmd and type in git checkout 1.8.4)
+	
+Then follow the steps from WinObjC github repo documentation to create a visual studio solution.
+
+In Visual Studio expand the solution and navigate to Bolts folder. Then right click on Bolts target go to properties->clang->Enable ObjectiveC ARC and set it to YES(-fobjc-arc).Do this for the other target BoltsInClassPath also in subsequent BoltsInClassPath folder.
+
+In the BoltsInClassPath sample app to make this app run on Windows I have found two issues in the app build phase.
+
+1) When you start building app you may encounter
+ Bolts.lib (BFExecutor_F456FB2F.obj) : error LNK 2019: unresolved external symbol _pthread_get_stackaddr_np referenced in function remaining_stack_size
+ Bolts.lib (BFExecutor_F456FB2F.obj) : error LNK 2019: unresolved external symbol _pthread_get_stacksize_np referenced in function remaining_stack_size
+
+This happens because I see a different implementations for pThread on Winobjc. so my temp workaround is
+
+go to your Bolts dependency in the solution and navigate to Bolts/Bolts/Common/BFExecutor.m file. change the below method implementation like this.
+
+__attribute__((noinline)) static size_t remaining_stack_size(size_t *restrict totalSize) {
+
+    pthread_t currentThread = pthread_self();
+
+
+	// NOTE: We must store stack pointers as uint8_t so that the pointer math is well-defined
+
+	uint8_t *endStack = 0;//pthread_get_stackaddr_np(currentThread);
+
+	*totalSize = 0;//pthread_get_stacksize_np(currentThread);
+
+
+	// NOTE: If the function is inlined, this value could be incorrect
+
+	uint8_t *frameAddr = __builtin_frame_address(0);
+
+
+	return (*totalSize) - (endStack - frameAddr);
+
+}
+
+
+2) go to your Bolts dependency in the solution and navigate to Bolts/Public Headers/Bolts.h file. change the line in import statements like this.
+	#if __has_include(<Bolts/BFAppLink.h>) && WINOBJC && !TARGET_OS_WATCH && !TARGET_OS_TV
```

Comment:
this is being displays as #if __has_include() && WINOBJC && !TARGET_OS_WATCH && !TARGET_OS_TV by GitHub markdown, so the Bolts/BFAppLink.h> is missing. Putting this between GitHub markdown tags for code ​
```Objective-C ... ``` should fix it.